tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetWebApi.cs

```diff
@@ -60,6 +60,7 @@ public abstract class AspNetWebApi : AspNetBase, IClassFixture<IisFixture>
         public AspNetWebApi(IisFixture iisFixture, ITestOutputHelper output, bool classicMode, bool enableSecurity)
             : base("WebApi", output, "/api/home/shutdown", @"test\test-applications\security\aspnet")
         {
+            SetEnvironmentVariable(Configuration.ConfigurationKeys.DebugEnabled, "true");
```

Comment:
do we want debug log level for the integration tests? might slow them down a bit?

Reply:
fixed in 742b004d7